src/darwin/Framework/CHIP/templates/templates.json

```diff
@@ -91,6 +91,16 @@
             "path": "CHIPAttributeTLVValueDecoder-src.zapt",
             "name": "Decode TLV attribute values into Objc objects",
             "output": "src/darwin/Framework/CHIP/zap-generated/CHIPAttributeTLVValueDecoder.mm"
+        },
+        {
+            "path": "../../../../../examples/chip-tool-darwin/templates/CHIPTestClustersObjc.zapt",
```

Comment:
examples/chip-tool-darwin/templates/templates.json seems to alraedy contain these. Why this change?

Reply:
I thought this was needed to fix an issue with codegen that I ran into, but as discussed in this [slack conversation](https://csamembers.slack.com/archives/G014G30SVV0/p1646935222327169), this won't be needed anymore.